CMakeLists.txt

```diff
@@ -122,11 +123,13 @@ if(FCL_WITH_OCTOMAP)
           set(OCTOMAP_LIBRARIES "octomap;octomath")
       endif()
   endif()
-  if (OCTOMAP_FOUND OR (OCTOMAP_INCLUDE_DIRS AND OCTOMAP_LIBRARY_DIRS))
-    string(REPLACE "." ";" VERSION_LIST ${OCTOMAP_VERSION})
-    list(GET VERSION_LIST 0 OCTOMAP_MAJOR_VERSION)
-    list(GET VERSION_LIST 1 OCTOMAP_MINOR_VERSION)
-    list(GET VERSION_LIST 2 OCTOMAP_PATCH_VERSION)
+  if(OCTOMAP_INCLUDE_DIRS AND OCTOMAP_LIBRARY_DIRS AND OCTOMAP_VERSION)
+    if(NOT OCTOMAP_MAJOR_VERSION AND NOT OCTOMAP_MINOR_VERSION AND NOT OCTOMAP_PATCH_VERSION)
```

Comment:
I think [the upstream change #137](https://github.com/OctoMap/octomap/pull/137) should be merged first because we couldn't reach here without `OCTOMAP_VERSION`. Even it's merged this still wouldn't work if an octomap without the change is installed.

Possible option I can think of now is we enforce the minimum required octomap version to 1.8.1 (or the minimum version that includes the version variable) for CMake config mode like: `find_package(octomap 1.8.1 QUIET)` .